### DISCUSSION TITLE
Link my account to a second E-mail

### DISCUSSION THREAD

**Discussion by ADNANE-RBT** [community/community#40722] [New to GitHub] [answered]
I want to link my GitHub account to a second e-mail, something like a recovery e-mail in case I lost my first e-mail.
is there a way to do it ? or is it that the moment i loose my e-mail i immediately loose the GitHub access too ?

**Comment by mohamedzhioua** [comment, marked as answer]
hello @ADNANE-RBT  you can add a new email  and the answer gonna be in the below link : " https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account"
or you can secure your account following the steps here : "https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials"

**Reply by ADNANE-RBT** [reply]
thanks a lot. im following the steps to secure my account

**Reply by mohamedzhioua** [reply]
with pleasure brother

**Comment by Gitana1616** [comment]
The second email would be thefloyd77@gmail.com my husband or my son taivenashur8213@gmail.com don't know how to add

**Comment by memoryseven** [comment]
我将尽快给您回复。
